### PR TITLE
LPS-116458 Removed unused style and function

### PR DESCRIPTION
**Pull request by brunofariasdeo**
https://issues.liferay.com/browse/LPS-116025

### CODE REVIEW

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:test:forms

**Comment by brunofariasdeo** [comment]
ci:test:forms

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_share_url_popover.scss [hunk 1]
```diff
@@ -1,14 +1,6 @@
 $c: '.ddm-share-url-popover';
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_share_url_popover.scss:1]: @brunofariasdeo, from what I remember, we realized that this class wasn't used too, right? Can you please double check? If this class isn't being used too, please remove it.

Comment (natocesarrego) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_share_url_popover.scss:1]: Hey @brunofariasdeo, have you had the chance to check my comment above? Thanks.

Comment (brunofariasdeo) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/css/form-builder/_share_url_popover.scss]: Hi @natocesarrego, as discussed, I've deleted the css file once ddm-share-url-popover is not being used.

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:test:forms

**Comment by brunofariasdeo** [comment]
ci:test:sf

**Comment by brunofariasdeo** [comment]
ci:test:relevant

**Comment by brunofariasdeo** [comment]
ci:forward

**Pull request closed without merging**